content/src/service/Service.ts

```diff
@@ -93,3 +101,5 @@ export type DeploymentEvent = {
 }
 
 export type DeploymentListener = (deployment: DeploymentEvent) => void | Promise<void>
+
+export type ErrorList = string[]
```

Comment:
As I said before, something like `InvalidDeploymentResult` would be a better name, and we should probably make this an object like:
```
{ errors: string[] }
```

That way, we can add more information in the future if we need to, without having to make changes